.github/workflows/check-pr-semantic-title.yml

```diff
@@ -0,0 +1,26 @@
+# This workflow will ensure that PR is titled according to semantic commits convention, so we can later create releases
+
+name: Check PR semantic title
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+    branches:
+      - master
```

Comment:
I guess we would want to keep semantic commits for all PRs, also if they are not targeted to get into `master`